Jenkinsfile

```diff
@@ -26,7 +26,8 @@ pipeline {
     }
     environment {
         DOCKER_TAG = "${env.BRANCH_NAME}_${getSha()}"
-        CA_STYLEGUIDE_VERSION_TAG = "${DOCKER_TAG.toLowerCase()}"
+        // Using the commit SHA would mean that every build gets a different tag and this busts the cache between PR builds.
+        CA_STYLEGUIDE_VERSION_TAG = "${env.BRANCH_NAME}"
```

Comment:
Does this mean that 'master' will always be the same version forever? I'm not saying that's a bad thing - it's pretty common practice in Docker, iirc.

Reply:
No, there's a dev_$containername_master tag which will get updated to the latest build on master after each setup step.

Since it's not a project released through Docker and just a build cache, there's not much call for keeping old versions around.